apollo-router/src/plugins/telemetry/config_new/events.rs

```diff
@@ -244,49 +195,39 @@ where
     Attributes: Selectors<Request, Response, EventResponse> + Default,
     Sel: Selector<Request = Request, Response = Response> + Debug,
 {
-    pub(super) request: StandardEvent<Sel>,
-    pub(super) response: StandardEvent<Sel>,
-    pub(super) error: StandardEvent<Sel>,
+    pub(super) request: Option<StandardEvent<Sel>>,
+    pub(super) response: Option<StandardEvent<Sel>>,
+    pub(super) error: Option<StandardEvent<Sel>>,
     pub(super) custom: Vec<CustomEvent<Request, Response, EventResponse, Attributes, Sel>>,
```

Comment:
Also it removes useless code with custom events, because `custom` is filtered to not have events that are `Off` but because `EventLevel` contained this value it resulted in useless checks